languages/ruby/bitwarden_sdk_secrets/lib/extended_schemas/schemas.rb

```diff
@@ -20,6 +21,7 @@ def to_dynamic
           "sync"             => sync&.to_dynamic,
           "secrets"          => secrets&.to_dynamic,
           "projects"         => projects&.to_dynamic,
+          "generators"       => generators&.to_dynamic,
```

Comment:
can we add this to README and optionally example ?

Reply:
We actually aren't exposing the generators functionality here. This was added to prevent `dry-type` from throwing errors when it parsed our schema file.

Reply:
Got a different vibe when i looked at this, make sense now! 👍